apply nomalization to rgb data along the time dim

zxdawn opened the pull request:
See this discussion: https://github.com/aazuspan/wxee/discussions/76

zxdawn committed:
apply nomalization to rgb data along the time dim

aazuspan reviewed (changes requested):
Thanks @zxdawn! The failed check is unrelated to your change, but I ran tests locally and `test_normalize` is failing. That test (below) uses a DataArray without a `variable` dimension, which should be supported. 

https://github.com/aazuspan/wxee/blob/1c4159384ff69095ace6dc475360a2f6a43b336a/test/test_xarray.py#L60-L68

Instead of specifying which dims to use, it will probably be safer to just use everything *but* `time`. Something like this should work:

```python
reduce_dims = set(da.dims) - {"time"}
min_val = da.quantile(1 - stretch, dim=reduce_dims)
```

zxdawn commented:
Thanks for review. I will update the PR soon. Sorry for the delay!

aazuspan committed:
Normalize on all dimensions but time

The "variable" dimension may not exist, so we should dynamically
set the reduced dimensions to everything but time.

aazuspan reviewed (approved):
@zxdawn, I'm going to make a release shortly and wanted to include your fix, so I went ahead and made that last change. Thanks for finding the issue and suggesting the fix!
